libs/server/Resp/RespServerSession.cs

```diff
@@ -299,6 +305,9 @@ public override int TryConsumeMessages(byte* reqBuffer, int bytesReceived)
             }
             finally
             {
+                // reset the session's flag for AOF blocking to default value after processing all commands
+                waitForAofBlocking = false;
```

Comment:
If above comment makes sense, then this is no longer necessary.

Reply:
Looking at how the RespSession objects are created, I see that they're held in Network Handler, and may be reused. So it makes sense to set the state of the session back to default value for the flag, right?

you would still want to reset the session flag at the very end of all commands, since the last command may set it to block OnAof and then after we exit command processing we may have a Session be left in state where the flag is set to true.

Reply:
It doesn't matter because the next time you receive a chunk of data, this line will fire and reset the bool. Because at that point there is no data yet injected into the response buffer.

`if (dcurr == networkSender.GetResponseObjectHead()) waitForAofBlocking = false;`